rag-sql-router/app.py

```
@@ -0,0 +1,462 @@
+                        }
+                        tool_usage_info.append(tool_info)
+            
+            # Simple formatting logic: if we have tool responses, format them
+            if tool_usage_info:
+                formatted_response = ""
+                for i, tool_info in enumerate(tool_usage_info, 1):
+                    # Determine if this is a document tool (anything not SQL)
+                    is_doc_tool = tool_info['tool_name'] != 'sql_tool'
+                    
+                    if is_doc_tool:
+                        formatted_response += f"**🔧 Tool Used:** `document_tool`\n\n"
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)